infra/modules/curi/mantarray_downloads/dns.tf

```diff
@@ -0,0 +1,71 @@
+resource "aws_route53_zone" "main" {
+  name = var.hosted_zone
+}
+
+resource "aws_route53_zone" "sub" {
```

Comment:
do we need a sub zone, or is it not that relevant when we're managing through tf?

Reply:
I removed the subdomain